lib/core/src/lib/card-view/components/card-view-selectitem/card-view-selectitem.component.html

```diff
@@ -17,7 +17,7 @@
                     {{ property.displayValue | async | translate }}
                 </div>
                 <div *ngIf="isEditable">
-                    <mat-form-field class="adf-property-value">
+                    <mat-form-field class="adf-property-value adf-form-field-input">
```

Comment:
Why start appending styles ? maybe it would be better have a specific styles that we want to apply on this element to avoid side effects